docs/cluster-management/zip-artifacts.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: main
+title: Setting Up a Zip Artifacts
+category: Cluster Management
+menu: menu
+toc:
+- title: Setting Up Zip Artifacts
+  url: "#setting-up-zip-artifacts"
+  active: 'true'
+- title: Architecture
+  url: "#architecture"
+- title: Setup
+  url: "#setup"
+- title: Setup artifacts-unzip-service
+  url: "#setup-artifacts-unzip-service"
+- title: Enable feature flags in API
+  url: "#enable-feature-flags-in-api"
+- title: When unzip failed
+  url: "#when-unzip-failed"
+---
+
+# Setting Up Zip Artifacts
+
+Screwdriver can upload build artifacts as a zip file using `screwdriver-artifact-bookend`.  
+This feature reduces the execution time of `screwdriver-artifact-bookend`.  
+It will takes time for the uploaded files to appear on UI, because the ZIP file is uploaded and then unzipped.
+
+## Architecture
+
+![zip artifacts architecture](../../cluster-management/assets/zip-artifacts-architecture.png)  
+
+1. Build container(`screwdriver-artifact-bookend` step) uploads a zipped build artifact.
+1. Build container sends request to API to unzip the zipped file.
+1. API sends the unzip message to Redis Queue.
```

Comment:
https://github.com/screwdriver-cd/guide/pull/527#discussion_r841304253 is not applied in English ver.

Reply:
I fixed not to use Redis Queue